generators/server/templates/src/main/java/package/service/LoginAlreadyUsedException.java.ejs

```diff
@@ -0,0 +1,27 @@
+<%#
+ Copyright 2013-2019 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+package <%=packageName%>.service;
+
+public class LoginAlreadyUsedException extends RuntimeException {
```

Comment:
UsernameAlreadyusedException seems like a better name because you don't have to guess what login means.

Reply:
Can do, just used the names we have already in `web/errors` package